keyboards/eggman/config.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2021 qpockets
```

Comment:
Best to use a real name here (either a person or a legal business entity) here, in addition to the userid.

Reply:
hey thanks for the review! 
as for the requested changes, is this absolutely necessary? My previous PR used my github user-name (which is also my username almost everywhere) and would prefer to stay consistent with that.

Reply:
We probably just didn't notice it on the other PR...
Ideally, it should be changed in both places.

We really do need to be careful about these things, because those notices are what keeps unscrupulous companies from taking your code, and using it for profit in a close-source way without sharing the profit with you.

Reply:
fair enough, changes committed. thanks again